eng/mgmt/automation/generate.py

```diff
@@ -343,7 +363,8 @@ def main():
     if succeeded:
         succeeded = compile_arm_package(sdk_root, module)
         if succeeded:
-            compare_with_maven_package(sdk_root, service, stable_version, current_version, module)
+            latest_release_version = get_latest_release_version(current_version, stable_version)
```

Comment:
Why call this on `stable_version`?

Reply:
Right, thanks... Didn't test codegen.
Fixed and test passed in local generation.